package.json

```diff
@@ -23,80 +23,36 @@
   },
   "license": "MIT",
   "scripts": {
+    "setup": "npm install && npm run validate -s",
+    "validate": "kcd-scripts validate",
     "prepare": "npm run build",
-    "build": "babel --out-dir lib src",
-    "format": "prettier-eslint --write \"**/*.{js,ts,json,yml,md,mdx}\"",
+    "build": "kcd-scripts build",
+    "lint": "kcd-scripts lint",
     "coverage": "codecov",
-    "test": "jest",
-    "test:ci": "npm run test && npm run coverage",
+    "test": "kcd-scripts test",
     "docs:dev": "docz dev",
     "docs:build": "docz build",
     "contributors:add": "all-contributors add"
   },
   "dependencies": {
-    "@babel/runtime": "^7.5.4",
-    "@types/testing-library__react-hooks": "^3.4.0"
+    "@babel/runtime": "^7.12.5"
   },
   "devDependencies": {
-    "@babel/cli": "7.12.8",
-    "@babel/core": "7.12.9",
-    "@babel/plugin-proposal-object-rest-spread": "7.12.1",
-    "@babel/plugin-transform-modules-commonjs": "7.12.1",
-    "@babel/plugin-transform-runtime": "7.12.1",
-    "@babel/preset-env": "7.12.7",
-    "@babel/preset-react": "7.12.7",
+    "@types/jest": "^26.0.15",
     "all-contributors-cli": "6.19.0",
     "babel-eslint": "10.1.0",
```

Comment:
Does this need to be explicitly listed still?

Reply:
right we don't need yet